CI_YAML.md

```diff
@@ -124,33 +125,155 @@ following are a list of keys that are reserved for special use.
 **Properties** is a Map<String, String> and any special values must be JSON encoded
 (i.e. no trailing commas). Additionally, these strings must be compatible with YAML multiline strings
 
-**$flutter/osx_sdk**: xcode configs including sdk and runtime. **Note**: support on legacy `xcode`/`runtime`
-properties and `xcode` dependency has been deprecated.
+<table>
```

Comment:
No change requested but doesn't github support markdown?

Reply:
It does, but it supports only simple case.
For a table containing code blocks (like yaml block here), we have to use html language.